<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_ui</class>
 <widget class="QWidget" name="main_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="img_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;resource\exicom_logo.png&quot; width=&quot;120&quot; height=&quot;40&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ref_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(240, 240, 240);
border:1px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 15pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../resource/reload.png</normaloff>../resource/reload.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>16</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="data_table">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>600</width>
     <height>180</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QComboBox" name="part_combo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>150</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(240, 240, 240);
border:1px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 10pt &quot;Times New Roman&quot;;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="table_combo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>148</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(240, 240, 240);
border:1px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 15pt &quot;Times New Roman&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="export_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(240, 240, 240);
border:1px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 15pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="graph_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(240, 240, 240);
border:1px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 15pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="fault_table">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>600</width>
     <height>290</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
